workers/data_refinery_workers/processors/salmon.py

```diff
@@ -726,18 +726,34 @@ def _get_tximport_inputs(job_context: Dict) -> Dict[Experiment, List[ComputedFil
         if should_run_tximport:
             quant_files = []
             for result in salmon_quant_results:
-                quant_files.append(ComputedFile.objects.filter(result=result, filename="quant.sf")[0])
+                try:
+                    quant_files.append(ComputedFile.objects.filter(result=result, filename="quant.sf")[0])
+                except:
+
+                    logger.exception(
+                        "Salmon quant result found without quant.sf ComputedFile!",
+                        processor_job=job_context["job_id"],
+                        quant_result=result.id,
+                        experiment=experiment.id
+                    )
+                    job.failure_reason = (
+                        "Salmon quant result {} found without quant.sf"
+                        " ComputedFile in experiment {}!"
+                    ).format(str(result.id), str(experiment.id))
```

Comment:
Is there anyway we can get the sample accession here? This is going to be harder to investigate this way.